spec/requests/users_spec.rb

```diff
@@ -2,6 +2,32 @@
 
 RSpec.describe 'Users', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    it 'returns http success' do
+      get '/users/index'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      user = User.create(name: 'Austin ')
+      get user_path(user)
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'Elements for users' do
+    it 'render the template of post index' do
+      get '/users'
+      expect(response.body).to render_template('index')
+      expect(response.body).to include('<h1>Show all users</h1>')
+    end
+
+    it 'render the show template for post' do
```

Comment:
- [ ] Kindly fix your failing specs. You need to install the `rails-controller-testing` gem as indicated by the error message.

  ![image](https://github.com/Obimbo07/tdn/assets/81468675/2eff75fb-5e7b-40f3-87f7-a7991da1ef5e)